packages/core-utils/src/itinerary.ts

```diff
@@ -521,8 +521,10 @@ export function calculateEmissions(
   const totalCarbon =
     itinerary?.legs?.reduce((total, leg) => {
       return (
-        (leg.distance * carbonIntensityWithDefaults[leg.mode.toLowerCase()] ||
-          0) + total
+        // We estimate that the actual driving distance is 1.34x the "crow-flying" distance
+        (leg.distance *
+          1.34 *
```

Comment:
should this maybe be a const at the top of the file? Please at least add a comment explaining where it comes from

Reply:
I'm weary of making it a const since I pretty much just made it up. It feels kinda right based on how "off" the values were